docs/getting-started/endgame-sensor-FDA.asciidoc

```diff
@@ -45,15 +50,17 @@ image::images/fda/allow-kernel-ext.png[Allow kernel extension]
 If the prompt does not appear, enable the extension by doing the following:
 
 . Open a Terminal application.
-. Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary.
-You should receive an output similar to `149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`. If you receive this output, the kernel extension is enabled.
-
+. Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary. You will receive an output that's similar to the one below:
++
+`149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`
++
+If you receive this output, the kernel extension is enabled.
 
 [discrete]
 [[system-extension]]
 == Approve the system extension for the Endgame sensor
 
-To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endgame with system extensions, you must enable Full Disk Access for the system extension during installation on macOS Big Sur (11.0) and later.
+To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endgame with system extensions, you must enable the system extension during installation on macOS Big Sur (11.0) and later.
```

Comment:
Notes for the steps in this section:
Update step 3 to: Click **Allow** to allow the ElasticEndpoint system extension to load.

Update step 4: Enter your username and password and click Modify to save changes.

Remove steps 5-7. 

Note to self: Ricardo will provide screenshots after I incorporate text changes.

Reply:
Addressed in [8e49b46](https://github.com/elastic/security-docs/pull/3565/commits/8e49b46eddaf7a73a9cb2465506e1ebefd28bc62).